react/features/invite/components/dial-in-summary/native/DialInSummary.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+import React, { Component } from 'react';
+import { type Dispatch } from 'redux';
+import { Linking, View } from 'react-native';
+import { WebView } from 'react-native-webview';
+
+import { translate } from '../../../../base/i18n';
+import {
+    BackButton,
+    Header,
+    HeaderLabel,
+    SlidingView
+} from '../../../../base/react';
+import { connect } from '../../../../base/redux';
+
+import { setDialInSummaryVisible } from '../../../actions';
+import { getDialInfoPageURLForURIString } from '../../../functions';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * The URL to display the summary for.
+     */
+    _summaryUrl: ?string,
+
+    dispatch: Dispatch<any>
+};
+
+/**
+ * Implements a React native component that displays the dial in info page for a specific room.
+ */
+class DialInSummary extends Component<Props> {
+
+    /**
+     * Initializes a new instance.
+     *
+     * @inheritdoc
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this._onCloseView = this._onCloseView.bind(this);
+        this._onNavigate = this._onNavigate.bind(this);
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const { _summaryUrl } = this.props;
+
+        return (
+            <SlidingView
+                position = 'bottom'
+                show = { Boolean(_summaryUrl) } >
+                <View style = { styles.webViewWrapper }>
+                    <Header>
+                        <BackButton onPress = { this._onCloseView } />
+                        <HeaderLabel labelKey = 'info.label' />
+                    </Header>
+                    <WebView
```

Comment:
We should handle onError here: https://github.com/react-native-community/react-native-webview/blob/master/docs/Reference.md#onerror

Reply:
Also, what are we showing while it loads? Maybe we should have a spinner covering it